commons-dlp/src/main/resources/db/changelog/exo-dlp.db.changelog-1.0.0.xml

```diff
@@ -66,4 +66,15 @@
     </addColumn>
   </changeSet>
   
+  <changeSet author="exo-dlp" id="1.0.0-8">
```

Comment:
Why not -6 ?

Reply:
because it comes after another PR in feature/ma which have already define 1.0.0-6 and 1.0.0-7.
I wait for this PR before merging mine

Reply:
OK thanks for the explanation